README.md

```diff
@@ -2,29 +2,27 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-cdm-cloudtrail-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/cool-cdm-cloudtrail-tf-module/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This Terraform module creates the necessary resources to allow the CDM
+Splunk user to access and import the CloudTrail logs for an account.
+In order to ensure that the CDM Splunk user cannot cause any harm a
```

Comment:
```suggestion
In order to ensure that the CDM Splunk user cannot cause any harm, a
```

Reply:
Normally I would put a comma here, but I didn't here because I think it reads awkwardly with the comma-delimited list of resource types following so closely afterwards.

Reply:
Fair enough- I ain't dying on this mole hill.